src/theme/stories/sizeShowcase.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+import { Td, Tr, useTheme } from '@chakra-ui/react';
+
+type Props = {
+  name: string;
+};
+
+const SizingShowCase: FC<Props> = ({ name }) => {
```

Comment:
To avoid manual update whenever Spaces/Sizes or any theme element changes, I would recommend the component should handle querying necessary data by itself.

```ts
const SizingShowCase: FC<Props> = () => {
  const { size } = useTheme();
  const { gray100 } = useToken('colors', 'gray.100');
  return (
    <>
      {Object.entries<string>(size).map(([key, value]) => {
        const unitRem = parseFloat(value);
        const unitPixel = unitRem / 0.0625;
        return (
          <Tr key={key} border="1px" borderColor={gray100}>
            <Td>{key}</Td>
            <Td>{`${unitRem}rem`}</Td>
            <Td>{`${unitPixel}px`}</Td>
          </Tr>
        );
      })}
    </>
  );
};
```

If we agree on this point, we can also apply same behavior for other tokens

Reply:
Nice suggestion. We can check that approach and apply it latter on. Thanks!